### ISSUE TITLE
Error Uploading Megapirate V3.0.1 to Hobbyking Multiwii and Megapirate AIO Flight Controller w/FTDI (ATmega 2560) V2.0

### ISSUE THREAD

**Issue by obbila** [MegaPirateNG/ardupilot-mpng#89]
Hi Apologies if this it not the correct place for this question but I am trying to upload Megapirate V3.0.1 to a Hobbyking Multiwii and Megapirate AIO Flight Controller w/FTDI (ATmega 2560) V2.0.

I can verify the code in Arduino and this works out fine but when I try to upload the firmware to the board I get the following message: avrdude: stk500v2_ReceiveMessage(): timeout

The RX light does flash during this time so I think I am connected to the correct port.

I am trying to do this on Mac OSX Mavericks

Thanks

**Comment by wengwah** [comment]
You have to use the special arduino compiler and follow the instructions provided.

**Comment by SirAlex** [comment]
> avrdude: stk500v2_ReceiveMessage(): timeout

You must check COM port you are selected in Arduino IDE.

**Comment by obbila** [comment]
Hi Thanks for you comments. I have found out what the issue was. Apple have released their own FTDI drivers as part of Mavericks. These cause problems with serial ports. The reason I was confused is because I could see that my device was connected in the Arduino IDE but no communication was happening. I found the fix here (http://forum.arduino.cc/index.php?topic=198539.0). It requires deactivating the Mavericks FTDI drivers and installing the chip manufacture's Drivers. 

I did this and have just successfully uploaded Megapirate to the flight controller board!

Thanks again!
